pages/_/marketing-candidate/download.js

```diff
@@ -0,0 +1,225 @@
+import * as React from "react";
+import * as Constants from "~/common/constants";
+import * as Actions from "~/common/actions";
+import * as System from "~/components/system";
+
+import { css } from "@emotion/react";
+
+import WebsitePrototypeWrapper from "~/components/core/WebsitePrototypeWrapper";
+import WebsitePrototypeHeader from "~/components/core/NewWebsitePrototypeHeader";
+import WebsitePrototypeFooter from "~/components/core/NewWebsitePrototypeFooter";
+
+const STYLES_ROOT = css`
+  padding: 16px 88px;
+  section {
```

Comment:
There is a ton of potential for side effects here. Please do not do this anymore (styling ANY of the tags like this)

I've remarked about this to @tarafanlin as well. I'm going to let it slide for the sake of progress but will make you both come back and clean this up right after V1 launch.

Reply:
Will refactor.